slurp/src/main/webapp/scripts/login.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const firebaseUiConfig = {
+  signInSuccessUrl: 'view-trips.html',
+  signInOptions: [
+    // This app requires a Google account to use.
+    firebase.auth.GoogleAuthProvider.PROVIDER_ID
+  ]
+};
+
+const firebaseAuthUi = new firebaseui.auth.AuthUI(firebase.auth());
+firebaseAuthUi.start('#firebaseui-login', firebaseUiConfig);
```

Comment:
out of curiosity, where does #firebaseui-login lead to?

Reply:
+1. is there a screenshot available for these pages?

Reply:
Added a GIF showing the process. 👍